src/main/kotlin/com/hibob/types/properties.kt

```diff
@@ -0,0 +1,49 @@
+package com.hibob.types
+
+import java.time.DayOfWeek
+
+// task 1
+class Store (val day: DayOfWeek, val products: List<Product>) {
+
+    // task 2
+    val open: Boolean
```

Comment:
I would call it in more indicate name, like isStoreOpen

Reply:
changed it, thanks